### PR TITLE
Password Reset functionality 

### PR DESCRIPTION
**Pull request by arpit456jain**
## Related Issuse

Password Reset functionality 

Closes: #115

#### Describe the changes you've made

https://user-images.githubusercontent.com/55352601/118126822-7d55fe80-b416-11eb-9d8d-1545290c6660.mp4

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by arpit456jain** [comment]
@khannakshat7 please merge this PR

**Comment by arpit456jain** [comment]
@CodesbyUnnati kindly review this PR

File: electricity/views.py [hunk 1]
```diff
@@ -15,7 +15,20 @@
...
+import random
+import math
 # Create your views here.
+def send_email_to_user(otp,email):
+    import smtplib
+    con = smtplib.SMTP("smtp.gmail.com",587)
+    con.ehlo()
+    con.starttls()
+    admin_email = "arpit456jain@gmail.com"
+    admin_password = "#jain vanshika#"
```
Comment (khannakshat7) [inline electricity/views.py:27]: Bro pl  remove ur password from this commit

**Comment by khannakshat7** [comment]
@arpit456jain  Awesome Work dude!!!

**Pull request merged**